src/__test__/Calculator.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
+import Calculator from '../components/Calculator';
+import '@testing-library/jest-dom';
+
+test('expect home component to match snapshot', () => {
+  const view = renderer.create(<Calculator />).toJSON();
+  expect(view).toMatchSnapshot();
+});
+
+test('renders button tag', () => {
+  render(<Calculator />);
+  const linkElement = screen.getByRole('button', { name: 'AC' });
+  expect(linkElement).toBeInTheDocument();
+});
+
+test('renders Ac in the UI', () => {
+  render(<Calculator />);
+  const linkElement = screen.getByText(/AC/i);
+  expect(linkElement).toBeInTheDocument();
+});
+
+test('renders Ac in the UI', () => {
+  render(<Calculator />);
+  const linkElement = screen.getByText('7');
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] Great work on implementing the snapshots so far :clap: but I guess you missed the point that the previous reviewer was making. He was saying you should test interactions such as button clicking and after the button clicking your calculator screen will show the output or something like that. Kindly make the tests for interactions.

**Hint:** You can use **import {fireEvent} from '@testing-library/react'** for this purpose :+1: You can also check this [documentation](https://testing-library.com/docs/dom-testing-library/api-events/) about fireEvent to get the understanding on how to use it and why :+1: